Code2CollegeBankingProject/BankingUI/views.py

```diff
@@ -155,21 +159,21 @@ def changeaccountinfo(request):
                     "title": request.user.first_name,
                     "response": "Your new information has been saved",
                 }
-                return render(request, "changeaccountinfo.html", userinfo)
+                return redirect(changeaccountinfo, userinfo)
```

Comment:
These redirects might be causing the pages to break! The `render` worked just as well. Remember that redirects take you from one page to another. In this case, we are wanting to stay on the same page. So the code shouldn't be using a redirect here.

Is there a particular reason why the code should use `redirect` instead of `render`? I looked at https://stackoverflow.com/questions/39560175/redirect-to-same-page-after-post-method-using-class-based-views

but I wasn't sure why the poster wanted to redirect instead of render.